plugins/cameraserver/cameraserver.gradle.kts

```diff
@@ -2,12 +2,10 @@ description = """
 The bundled CameraServer plugin. This plugin provides data sources and widgets for viewing MJPEG streams from the WPILib CameraServer.
 """.trimIndent().trim()
 
-plugins {
-    id("edu.wpi.first.wpilib.versioning.WPILibVersioningPlugin") version "1.6"
-}
-
 dependencies {
-    api(group = "edu.wpi.first.cscore", name = "cscore-java", version = "+")
-    implementation(group = "edu.wpi.first.cscore", name = "cscore-jni", version = "+")
-    implementation(group = "edu.wpi.first.wpilib", name = "opencv", version = "3.2.0")
+    compile(group = "edu.wpi.first.cscore", name = "cscore-java", version = "+")
+    compile(group = "org.opencv", name = "opencv-java", version = "3.2.0")
+    compile(group = "org.jcodec", name = "jcodec-javase", version = "0.2.0")
+    compile(group = "edu.wpi.first.cscore", name = "cscore-jni", version = "+", classifier = "all")
+    compile(group = "org.opencv", name = "opencv-jni", version = "+", classifier = "all")
```

Comment:
I don't understand why you are using the `compile` configuration here. You shouldn't if you are using the `java-library` plugin.

Reply:
The JNI ones are runtime only, so these definitely should be runtime. These artifacts are huge (~100mb combined), and definitely not needed to just compile plugins.

Reply:
You should only use the `api` or `implementation` configurations.